cosette/core.py

```diff
@@ -31,13 +32,17 @@
 # %% ../00_core.ipynb 5
 empty = inspect.Parameter.empty
 
+# %% ../00_core.ipynb 6
+mk_msg = mk_msg
```

Comment:
This is a bit of a hack to ensure that `mk_msg` and `mk_msgs` are included in `__all__`. 

Let me know if there's a cleaner way to do this.

Reply:
Yes the official way is a special `_all_` variable (**NB**: that's just a single underscore; this is just for nbdev). `_all_ = ['mk_msg']`